bugsnag-android-core/src/main/java/com/bugsnag/android/Client.java

```diff
@@ -86,6 +85,8 @@
     final ClientObservable clientObservable = new ClientObservable();
     private PluginClient pluginClient;
 
+    final Notifier notifier = new Notifier();
```

Comment:
Can `bugsnag-android` notifier details be set in `Client` on init rather than baked into the `Notifier` object? I think it makes it slightly more like a normal data object.

Reply:
Our release script which bumps the version relies on the values being set in the class - unless you feel strongly I'll keep this as-is to reduce the amount of change.